pdb2sql/StructureSimilarity.py

```diff
@@ -142,16 +143,18 @@ def compute_lrmsd_fast(self, lzone=None, method='svd', check=True, name=['C', 'C
                 self.ref, resData, return_not_in_zone=True, name=name)
 
             if data_decoy_long.symmetric_difference(data_ref_long) != set():
-                raise ValueError(
-                    'Issue in the calculation of the l-rmsd')
+                res = data_decoy_long.symmetric_difference(data_ref_long)
```

Comment:
The error is already raised in the `check_residues` function

Reply:
indeed but calling check_residue is optional so I would still like to raise the error if enforce_residue_matching==True
We could also take the opportunity to improve the terrible error message that I crafted :) Something a bit more descriptive like what you wrote above :   "Residue numbering not identical in ref and decoy\n Set enforce_residue_matching=False to bypass this error."

Reply:
That part of the script was also optional (`if check == True`); we were checking the same thing twice under the same condition. Now `if check == True or enforce_residue_matching==True`,  `check_residues` is called, and returns an error (`enforce_residue_matching==True`) or a warning (`enforce_residue_matching==False`) if the ref and decoy residues do no match.